### PR TITLE
typos

### PR DESCRIPTION
**Pull request by gottacatchenall**

### CODE REVIEW

**Comment by gottacatchenall** [comment]
also theres a [paper out today](https://doi.org/10.1038/s41467-021-24149-x) which uses interaction frequency as a mechanism which could be added to the list of models at the beginning of paragraph 3

**Comment by TanyaS08** [comment]
> also theres a [paper out today](https://doi.org/10.1038/s41467-021-24149-x) which uses interaction frequency as a mechanism which could be added to the list of models at the beginning of paragraph 3

I mean go for it

File: manuscript.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ and time [@Poisot2015SpeWhy]. Different types of interactions vary in their
 intrinsic predictability (*e.g.* some fungal species engage in opportunistic
 saprotrophy [@Smith2017GroEvi], obligate parasites are more deterministic
 in their interactions than facultative parasites [@Poisot2013FacObl;
-@Luong2019FacPar]). In addition to this variance in predictability, networks
+@Luong2019FacPar]. In addition to this variance in predictability, networks
```
Comment (TanyaS08) [inline manuscript.md:28]: that should stay - it looks weird but its bracket pair is on line 25.

**Review by TanyaS08 — CHANGES_REQUESTED** [review]

File: manuscript.md (hunk 1 above)
Comment (TanyaS08) [reply manuscript.md:28]: sanity check me  on this one though.

**Comment by TanyaS08** [comment]
I have this weird track record of builds failing and then passing so hopefully if you push something new then everything should pass.

File: manuscript.md (hunk 1 above)
Comment (gottacatchenall) [reply manuscript.md:28]: yeah

**Pull request merged**